doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ###############
 
+Changes for 6.2.0
```

Comment:
This is a big version jump for such a small change. Is your expectation that 6.2.0 will include other changes, as well (such as removing SALPY support)?